typescript-generator-core/src/test/java/cz/habarta/typescript/generator/Jackson2ParserTest.java

```diff
@@ -132,7 +132,7 @@ static Jackson2Parser getJackson2Parser() {
 
     @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "kind")
     @JsonSubTypes({
-        @JsonSubTypes.Type(value = SubTypeDiscriminatedByName1.class, name = "SubType1"), // value from @JsonTypeName is used
+        @JsonSubTypes.Type(value = SubTypeDiscriminatedByName1.class, name = "SubType1"),
```

Comment:
not sure about this one

this is probably a breaking change, since @JsonTypeName would no longer have priority over @JsonSubTypes.Type, and all typenames would be included